packages/contracts/scripts/generate-markdown.js

```diff
@@ -37,19 +38,15 @@ const networks = {
 
 ### Predeploy contracts:
 |Contract|Address|
-|--|--|
-|OVM_ETH: | \`0x4200000000000000000000000000000000000006\`
-|OVM_L2StandardBridge: | \`0x4200000000000000000000000000000000000010\`
-|OVM_L2CrossDomainMessenger: | \`0x4200000000000000000000000000000000000007\`
-|OVM_L2ToL1MessagePasser: | \`0x4200000000000000000000000000000000000000\`
-|OVM_L1MessageSender: | \`0x4200000000000000000000000000000000000001\`
-|OVM_DeployerWhitelist: | \`0x4200000000000000000000000000000000000002\`
-|OVM_ECDSAContractAccount: | \`0x4200000000000000000000000000000000000003\`
-|OVM_SequencerEntrypoint: | \`0x4200000000000000000000000000000000000005\`
-|Lib_AddressManager: | \`0x4200000000000000000000000000000000000008\`
-|ERC1820Registry: | \`0x1820a4B7618BdE71Dce8cdc73aAB6C95905faD24\`
+|--|--|\n`
 
----
+  for (const predeploy in predeploys) {
+    if (predeploys.hasOwnProperty(predeploy)) {
```

Comment:
You can get rid of the `hasOwnProperty` check if you do:

```js
for (const [name, addr] in Object.entries(predeploys)) { ... }
```

Reply:
Sounds good! Took this suggestion down.